### ISSUE TITLE
parcel issue + missing plugin-proposal-class-properties

### ISSUE THREAD

**Issue by lifeart** [tomdale/glimmer-lite-donut#1]
![image](https://user-images.githubusercontent.com/1360552/63128856-bb628c00-bfbe-11e9-98e8-c24a997d0162.png)

```
{
    "presets": [
      "@babel/preset-env"
    ],
    "plugins": [
      "@babel/plugin-proposal-class-properties",
      "@glimmerx/babel-plugin-component-templates"
    ]
  }
```

**Comment by lifeart** [comment]
after missing dependency install I have:

![image](https://user-images.githubusercontent.com/1360552/63128969-067c9f00-bfbf-11e9-95d9-b37cf1be5657.png)

**Comment by lifeart** [comment]
[parcel-glimmer-donut.zip](https://github.com/tomdale/glimmer-lite-donut/files/3507295/parcel-glimmer-donut.zip)

**Comment by lifeart** [comment]
looks like we need Babel plugin for decorators support

**Comment by lifeart** [comment]
partially working version: [parcel-glimmer-donut.zip](https://github.com/tomdale/glimmer-lite-donut/files/3508250/parcel-glimmer-donut.zip)

(but, nested components won't work)

![image](https://user-images.githubusercontent.com/1360552/63146868-1c14b780-c005-11e9-897a-71fb0a4875d1.png)

tldr:

this work:

```js
import Component, { hbs } from '@glimmerx/component';
import Button from './Button';

export default class MyComponent extends Component {
    static template = hbs`
      <h1>Hello world!</h1>
      <Button  // fix it and it will broke app
    `
}
```

this not:

```js
import Component, { hbs } from '@glimmerx/component';
import Button from './Button';

export default class MyComponent extends Component {
    static template = hbs`
      <h1>Hello world!</h1>
      <Button />
    `
}
```

**Comment by lifeart** [comment]
solved!
